server/src/controllers/cars/getCarsById.ts

```diff
@@ -0,0 +1,12 @@
+import { Request } from 'express';
+import { getCarInfo } from '../../services';
+
+const getCarsById = async (req: Request) => {
+  const { id } = req.params;
```

Comment:
try to validate `id` to ensure it's a number, If not end response as a bad request